src/content/datachannel/filetransfer/js/test.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+// This is a basic test file for use with testling.
+// The test script language comes from tape.
+/* jshint node: true */
+var test = require('tape');
+
+var webdriver = require('selenium-webdriver');
+var seleniumHelpers = require('../../../../../test/selenium-lib');
+
+test('Filetransfer via Datachannels', function(t) {
+  var driver = seleniumHelpers.buildDriver();
+
+  driver.get('file://' + process.cwd() +
+      '/src/content/datachannel/filetransfer/index.html')
+  .then(function() {
+    t.pass('page loaded');
+    // Based on https://saucelabs.com/resources/articles/selenium-file-upload
+    return driver.findElement(webdriver.By.id('fileInput'))
+       .sendKeys(process.cwd() + '/index.html');
+  })
+  .then(function() {
+    // Wait for the received element to be displayed.
+    return driver.wait(function() {
```

Comment:
Checkout the [variant](https://github.com/webrtc/samples/commit/3ec2294cadc7e5323f4bb83abc956c3da1ab84e2) I used to fix the flake in the input-output test, yes it does not have a timeout but it does work more cases it seems than using a timeout (seems to trigger on false positives sometimes.).